elixir_phoenix/priv/repo/migrations/20150930050801_create_client.exs

```diff
@@ -0,0 +1,22 @@
+defmodule EboshiApiShootoutElixirPhoenix.Repo.Migrations.CreateClient do
+  use Ecto.Migration
+
+  def change do
+    create table(:clients) do
+      add :name, :string
+      add :address, :string
+      add :city, :string
+      add :state, :string
+      add :zip, :string
+      add :country, :string
+      add :email, :string
+      add :contact, :string
+      add :phone, :string
+      add :created_at, :datetime
+      add :updated_at, :datetime
+
+      timestamps
```

Comment:
I'm guessing here, since I don't know Elixir or Phoenix, but this looks very inspired by Rails migrations, and if it is at all similar, `timestamps` is shorthand for the `created_at` and `updated_at` lines, so I bet you could remove them. Could be wrong though.